README.md

```diff
@@ -50,8 +56,8 @@ accessors, though, and invalid values will be ignored.
 
 ### Required
 
- - **token:** String. Authorization token for the Logentries service.
- - **region**: Mandatory argument. The region of ingestion endpoint to be used. Examples: 'eu', 'us'
+ - **token:** String. Authorization token for the Rapid7 Insight Platform.
```

Comment:
Authorization token? Isn't this just a log token?

Reply:
yes it is, changed